lua/spec/neotest-go/init_spec.lua

```diff
@@ -190,6 +190,24 @@ describe("discover_positions", function()
             type = "test",
           },
         },
+        {
+          {
+            id = vim.loop.cwd() .. "/neotest_go/cases_test.go::TestAdd::test_three",
+            name = '"test three"',
+            path = vim.loop.cwd() .. "/neotest_go/cases_test.go",
+            range = { 44, 1, 48, 3 },
+            type = "test",
+          },
+          {
+            {
+              id = vim.loop.cwd() .. "/neotest_go/cases_test.go::TestAdd::\"test three\"::test_four",
```

Comment:
There is something wonky with the treesitter query, those quotes are unexpected to say the least